includes/llms.template.hooks.php

```diff
@@ -171,6 +171,10 @@
 
 add_action( 'llms_my_grades_course_table', 'lifterlms_template_student_dashboard_my_grades_table', 10, 2 );
 
+add_action( 'llms_view_order_information', 'llms_template_view_order_information', 10 );
+add_action( 'llms_view_order_actions', 'llms_template_view_order_actions', 10 );
+add_action( 'llms_view_order_transactions', 'llms_template_view_order_transactions', 10, 2 );
```

Comment:
I may be wrong due to the brain melting fever or the DayQuill, but shouldn't the number of arguments be 4?
```suggestion
add_action( 'llms_view_order_transactions', 'llms_template_view_order_transactions', 10, 4 );
```